docs/contributor/pocs/node-local-traffic/node-local-traffic.md

```diff
@@ -0,0 +1,43 @@
+# Service with local traffic policy
+
+The PoC tries to understand the behavior of the OTLP push service with [service internal traffic policy](https://kubernetes.io/docs/concepts/services-networking/service-traffic-policy/#using-service-internal-traffic-policy). The gateways
+in Kyma Telemetry are exposed via OTLP service eg: `telemetry-otlp-traces`. the
+applications running in the cluster will push the telemetry data to this service. This PoC tests the behaviour when this OTLP 
+push service is configured with `internalTrafficPolicy: Local`. Following criteria were tested:
+- If the data is only sent to the daemonSet running on the same node
+- If the daemonSet for some reason is not running then is the data sent to a daemonSet running on a different node
```

Comment:
```suggestion
- If the DaemonSet is not running, then is the data sent to a DaemonSet running on a different node
```
Is this a question? "Is it sent to another DaemonSet?" If not, change where the "is" is in the sentence.